src/main/java/com/alphawallet/attestation/IdentifierAttestation.java

```diff
@@ -31,7 +30,7 @@ public IdentifierAttestation(String identity, AttestationType type, AsymmetricKe
     super();
     super.setVersion(18); // Our initial version
     super.setSubject("CN=" + AttestationCrypto.addressFromKey(key));
-    super.setSignature(AttestationCrypto.OID_SIGNATURE_ALG);
+    super.setSigningAlgorithm(AttestationCrypto.OID_SIGNATURE_ALG);
```

Comment:
Wait, OID_SIGNATURE_ALG is currently set to "1.2.840.10045.2.1" which is `ecPublicKey` according to this: https://oidref.com/1.2.840.10045.2.1 - which should be a component of PublicKeyInfo instead of attestation's signing algorithm?

Reply:
I have not made any change to the OIDs for a very long time. Initially when we tried to make everything work with X509 (in particular with Java's own X509 code) I think this was one of the only reasonable parameters we could set that actually worked. But as you say it is not completely correct and the reason is that there is no OID for what we actually do. 
However since we are actually not currently verifying things in a smart contract there is no reason why the attestation must be signed with Keccak256 and secp256k1. 
I can create an issue investigating this further and perhaps add support for standard signatures if you think it is relevant?